Bearsampp/Bearsampp
# Changes to bootstra 5.2

to test:
run ant release
test full and lite ( at least ) and verify styling looks ok and font sizes are reasonable.
